### DISCUSSION TITLE
check authentication

### DISCUSSION THREAD

**Discussion by PouryaNoufallah96** [nuxt/nuxt#16799] [Questions]
### Environment

production

### Reproduction

https://github.com/PouryaNoufallah96/udsurvey

### Describe the bug

hello. 
i'm checking my jwt authentication api in a nuxt plugin and everything works just fine in development. but when I deploy project on netlify in production env it seems like my plugin or middleware not working . even when I'm logged in and check-auth API called successfully but middleware redirect me to the login page and then in login page my home page will render again after check-auth api finished. i really tested everything

**Comment by sevillaarvin** [comment]
Are there any error messages in the console? Perhaps the response of the server is different when you are developing?
